torch/testing/_internal/hop_db.py

```diff
@@ -75,6 +75,7 @@ def f2(x, y0, y1):
     "triton_kernel_wrapper_functional",
     "hints_wrapper",
     "foreach_map",
+    "aoti_call_delegate",
```

Comment:
Can you add a test for this? It's important (for export tests too!) that this is tested

Reply:
aoti_call_delegate accepts an AOTInductorEPModule, which is defined in fbcode only, so it's easier to test this in fbcode.
and I have switch PT2I tests to start consuming this HOP, see D68599208

Reply:
Fair enough, can you put a comment here explaining this?